### DISCUSSION TITLE
How do I use the AddIn?

### DISCUSSION THREAD

**Discussion by deonholt** [EvilHacker/BoxJoint#7] [Q&A]
Hi and thanx for what seems to be an excellent app. I have seen on YouTube a video someone made using the AddIn to make a box for his laser-engraver.
I managed to successfully install it on my Windows machine and I see it under the "Modify" tab.
I then created to rectangular bodies, which touched, since they share the same "border line". But, after 30 minutes of trying, I still cannot get a box joint. I have them on the same plane and level on the XY plane.
Is there someone who can please share a short video on how to use the addin? It will be a great help and I will really appreciate it.

Thanks very much.
![Box-Joint](https://github.com/user-attachments/assets/f2beaaaf-ad8b-4de9-a73c-6bdb82c8cdd2)

**Comment by EvilHacker** [comment]
This Add-In only handles angled [Butt Joints](https://en.wikipedia.org/wiki/Butt_joint) and coverts them into [Box Joints](https://en.wikipedia.org/wiki/Box_joint). The two bodies must contact at an angle, typically at 90°, but other reasonable angles will work too.

This [animation](https://github.com/EvilHacker/BoxJoint?tab=readme-ov-file#screen-shots) shows how the sides of a box arranged as 90° [Butt Joints](https://en.wikipedia.org/wiki/Butt_joint) are selected to convert them into [Box Joints](https://en.wikipedia.org/wiki/Box_joint).

This Add-In does not handle other [joint types](https://en.wikipedia.org/wiki/Joinery#Types_of_wood_joints) that would work with the flat end-to-end geometry you have.
